ct/mongodb.sh

```diff
@@ -28,7 +28,7 @@ function update_script() {
     header_info
     check_container_storage
     check_container_resources
-    if [[ ! -f /etc/apt/sources.list.d/mongodb-org-7.0.list ]]; then
+    if [[ ! -f /etc/apt/sources.list.d/mongodb-org-7.0.list && ! -f /etc/apt/sources.list.d/mongodb-org-8.0.list ]]; then
```

Comment:
If we allow to install both, shouldn´t we provide a way to keep 7 up to date or is it end of live anyway?

Reply:
7.0 is actual, many apps use 7.x 
this command says "not of both available"

Reply:
Yeah, and i did not read the code correcttly, sorry for that misahap.